test_regress/t/t_class_member_sens.pl

```diff
@@ -11,6 +11,7 @@
 scenarios(vlt => 1);
 
 compile(
+    verilog_flags2 => ["--no-std"],
```

Comment:
Why does this test need this?

Reply:
It's true that it didn't need it. I treated these tests collectively, but now I've reviewed all of them and removed the flag wherever it has no practical effect.